pyproject.toml

```diff
@@ -6,12 +6,12 @@ authors = ["Python Discord <info@pythondiscord.com>"]
 license = "MIT"
 
 [tool.poetry.dependencies]
-python = "3.11.*"
+python = "3.12.*"
 
 # See https://bot-core.pythondiscord.com/ for docs.
-pydis_core = { version = "11.0.1", extras = ["all"] }
+pydis_core = { version = "11.1.0a1", extras = ["all"] }
```

Comment:
This will be bumped to 11.1.0 when https://github.com/python-discord/bot-core/pull/211 is merged & released.